Xamarin.Forms.Platform.iOS/Renderers/ListViewRenderer.cs

```diff
@@ -386,9 +386,12 @@ async void OnScrollToRequested(object sender, ScrollToRequestedEventArgs e)
 					//iOS11 hack
 					if (Forms.IsiOS11OrNewer)
 					{
-						await Task.Delay(1);
+						InvokeOnMainThread(() =>
+						{
+							if(Control != null && !_disposed)
+								Control.ScrollToRow(NSIndexPath.FromRowSection(index, 0), position, e.ShouldAnimate);
```

Comment:
Just looking at the code, I think this breaks ScrollTo on iOS<11

Reply:
🤦‍♂